src/main/java/Duke.java

```diff
@@ -1,10 +1,97 @@
+import java.util.Scanner;
 public class Duke {
+
+    private static Task[] tasks = new Task[100];
+    private static int taskCount = 0;
+    
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        greet();
+
+        String userResponde;
+        Scanner in = new Scanner(System.in);
+        do {
+            userResponde = in.nextLine();
+            dukeResponde(userResponde);
+
+        }while(!userResponde.equals("bye"));
```

Comment:
It would be nice if there was a space between } and "while" so that it is easier to read.